common/FileSystem.cpp

```diff
@@ -271,6 +274,134 @@ std::wstring FileSystem::GetWin32Path(std::string_view str)
 	return ret;
 }
 
+bool FileSystem::CreateWin32IShellLink(const std::string_view path_suffix, const std::string_view launch_arguments,
+	const std::string_view description, const GUID& base_directory_ID)
+{
+	const auto check_failed = [&](const HRESULT hresult, const bool initialized = true) -> bool {
+		if (!FAILED(hresult))
+			return false;
+
+		_com_error err(hresult);
+		const TCHAR* error_message = err.ErrorMessage();
+		Console.ErrorFmt("Failed to create shortcut: {}",
+			fmt::format("{} [{:08X}]", StringUtil::WideStringToUTF8String(error_message), hresult));
+
+		if (initialized)
+			CoUninitialize();
+
+		return true;
+	};
```

Comment:
Is it just me or does this not capture anything?

If your lambda doesn't capture, maybe it should be a function and not a lambda.  At least use an empty capture list to make it clear.